src/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0 auto;
+}
+
+.snake {
+  display: flex;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  flex-wrap: wrap;
+  justify-content: center;
+  line-height: 300px;
+
+  text-align: center;
+  align-items: center;
+}
+
+.items {
+  width: calc(100% / 3);
+  height: 300px;
+}
+
+.snake_head {
+  background-color: #f00;
+}
+
+.snake_tail {
+  background-color: black;
+  flex-wrap: wrap-reverse;
+}
+
+.part_2 {
+  background-color: #c00;
+}
+
+.part_3 {
+  background-color: #900;
+}
+
+.part_4 {
+  background-color: #600;
+  order: 3;
+  flex-wrap: wrap-reverse;
+}
+
+.part_5 {
+  background-color: #300;
+  order: 2;
+}
+
+@media screen and (max-width: 750px) {
+  .items {
+    width: calc(100% / 2);
+  }
+
+  .snake {
+    flex-wrap: wrap;
+  }
+
+  .part_3 {
+    order: 2;
+  }
+
+  .snake_tail {
+    order: 3;
+  }
+
+  .part_4 {
+    order: 1;
+  }
+}
+
+@media screen and (max-width: 450px) {
+  .snake {
+    flex-wrap: nowrap;
+    flex-direction: column;
+  }
+
+  .items {
+    width: 100%;
+  }
```

Comment:
some of this rules is redundant

Reply:
When I remove something from this, it stops working as it should